components/supabase/sources/new-row-added/new-row-added.mjs

```diff
@@ -0,0 +1,110 @@
+import base from "../common/base.mjs";
+import {
+  DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+  ConfigurationError,
+} from "@pipedream/platform";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "supabase-new-row-added",
+  name: "New Row Added",
+  description: "Emit new event for every new row added in a table. [See documentation here](https://supabase.com/docs/reference/javascript/select)",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    ...base.props,
+    column: {
+      propDefinition: [
+        base.props.supabase,
+        "column",
+      ],
+      optional: true,
+    },
+    filter: {
+      propDefinition: [
+        base.props.supabase,
+        "filter",
+      ],
+      optional: true,
```

Comment:
```suggestion
      description: "The name of the table to watch for new rows.",
```

Reply:
Added a period at the end of the description to follow Google's Developer docs style guide.